master/buildbot/master.py

```diff
@@ -93,6 +93,8 @@ def __init__(self, basedir, configFileName="master.cfg", umask=None):
         self.reconfig_active = False
         self.reconfig_requested = False
         self.reconfig_notifier = None
+        self.change_services_state = False
```

Comment:
I'd be more in favour of picking is_reconfiguring as that's the primary use-case where this is happening.